AccessibilityInsightsForAndroidService/app/src/main/java/com/microsoft/accessibilityinsightsforandroidservice/FocusVisualizer.java

```diff
@@ -24,27 +24,11 @@ public FocusVisualizer(
     this.focusVisualizationCanvas = focusVisualizationCanvas;
   }
 
-  public void HandleAccessibilityRedrawEvent(AccessibilityEvent event) {
-    if (event.getEventType() == AccessibilityEvent.TYPE_WINDOW_CONTENT_CHANGED) {
-      return;
-    }
-
-    if (event.getEventType() == AccessibilityEvent.TYPE_WINDOW_STATE_CHANGED) {
-      this.resetVisualizations();
-      return;
-    }
-
-    if (event.getEventType() == AccessibilityEvent.TYPE_VIEW_SCROLLED) {
-      this.updateDrawingsWithNewCoordinates();
-      return;
-    }
-    if (event.getEventType() == AccessibilityEvent.TYPE_WINDOWS_CHANGED) {
-      this.resetVisualizations();
-      return;
```

Comment:
This is on the same vain of the `TYPE_WINDOW_STATE_CHANGED` discussion we had.  I think there is an even better case for `TYPE_WINDOWS_CHANGED` to reset visualizations.  If you're in the settings app -> accessibility and draw focus order there then hit the back button it takes you to a completely different view but the visualizations are staying there. This event fires on window change and could reset those visualizations.  I feel like hedging towards not leaving zombie visualizations is better? Just my intuition.  Worth talking to Jui about, but it doesn't currently match what I would expect the behavior to be.